scripts/qa_crowbarsetup.sh

```diff
@@ -1016,8 +1016,10 @@ EOF
 
     if [ -z "$NOINSTALLCLOUDPATTERN" ] ; then
         safely $zypper in -l -t pattern cloud_admin
+        $zypper al kernel-default # dup below breaks for kernel-default so skip it
```

Comment:
Would that not prevent `qa_crowbarsetup.sh` from upgrading the kernel?

Reply:
only for the dup which cannot do it anyway. and then we remove the lock again

Reply:
Thanks.